java-sdk/src/main/java/com/netflix/conductor/sdk/workflow/def/tasks/Javascript.java

```diff
@@ -105,6 +105,7 @@ public Javascript validate() {
             LOGGER.error("missing " + ENGINE + " engine.  Ensure you are running supported JVM");
             return this;
         }
+        scriptEngine.put("polyglot.js.allowHostAccess", true);
```

Comment:
Why do we need this?

Reply:
This is required to access Java objects bound as input. (Nashorn allows this by default, but GraalVM requires explicit setting for security reasons: https://www.graalvm.org/22.2/reference-manual/js/NashornMigrationGuide/#secure-by-default). You can  validate this by removing the line and running the tests: it will result in errors like this:  `TypeError: Cannot read property 'testKey1' of undefined`

This is equivalent to `HostAccess.ALL` (see https://www.graalvm.org/22.2/reference-manual/js/ScriptEngine/) which should be identical to the Nashorn behavior. 

An alternative would be to use `HostAccess.EXPLICIT` for greater security, but that would result in a breaking change.

To my understanding, workflows are considered part of the source code of the host application and can be trusted at the same level, thus this should not pose a security risk.